templates/hybrid-lvm/disko-config.nix

```diff
@@ -0,0 +1,75 @@
+# Example to create a bios compatible gpt partition
+{ lib, disks ? [ "/dev/sda" ], ... }: {
+  disk = lib.genAttrs disks (dev: {
+    device = dev;
+    type = "disk";
+    content = {
+      type = "table";
+      format = "gpt";
+      partitions = [
+        {
+          name = "boot";
+          type = "partition";
+          start = "0";
+          end = "1M";
+          part-type = "primary";
+          flags = ["bios_grub"];
+        }
+        {
+          type = "partition";
+          name = "ESP";
+          start = "1MiB";
+          end = "100MiB";
```

Comment:
```suggestion
          end = "500MiB";
```

Reply:
yeah why not. or should we put just the ESP on the partiton and /boot to /?